src/Core/ModulesApplication.php

```diff
@@ -87,6 +87,7 @@ private function loadCustomModule($module, $eventDispatcher)
     {
         if (!is_readable($module['path'] . DIRECTORY_SEPARATOR . self::CUSTOM_MODULE_BOOSTRAP_NAME)) {
             // TODO: stephen need to escape filename here.
+            // TODO: normal escaping will break windows. need normalized paths first. Do we have?
```

Comment:
Can I get a quick example by what we are looking for with "normalized paths" ?

Reply:
\root/dir\sub/file.php same issue in documents between window separator and linux.

Reply:
normalized would be `/root/dir/sub/file.php`

Reply:
Can't we just not use the DIRECTORY_SEPARATOR and just use `/` character always? I remember this being issue in past where DIRECTORY_SEPARATOR caused problems.

Reply:
Certainly easier to type:)